src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1,12 +1,45 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
 import { renderHook } from '@testing-library/react-hooks';
 
-import { useCourseEnrollmentUrl } from '../hooks';
+import { CourseContextProvider } from '../../CourseContextProvider';
+import { useCourseEnrollmentUrl, useEnrollData } from '../hooks';
 import { LICENSE_SUBSIDY_TYPE } from '../constants';
 
 jest.mock('../../../../config', () => ({
   features: { ENROLL_WITH_CODES: true },
 }));
```

Comment:
If this is now enabled by default, should we remove it while we're in here?

Reply:
will be removed once we enable in prod